frontend/components/site/SiteSettings/AdvancedSiteSettingsForm.jsx

```diff
@@ -103,7 +103,7 @@ AdvancedSiteSettingsForm.propTypes = {
   // initialValues is what the initial form values are based on
   initialValues: PropTypes.shape({
     engine: PropTypes.string.isRequired,
-    config: PropTypes.string,
+    defaultConfig: PropTypes.string,
```

Comment:
@amirbey checkout the `AdvancedSiteSettings` component feeding the props to this component.  I noticed it is still `config` and not `defaultConfig` props. https://github.com/18F/federalist/blob/config-jsonb/frontend/components/site/SiteSettings/AdvancedSiteSettings.jsx#L28-L33

Reply:
good catch @apburnes ... updated